workflow/rules/prepare.smk

```diff
@@ -225,16 +227,16 @@ rule get_exons_gtf:
 ###############################################################################
 
 
-rule gtftobed:
+rule exons_gtf_to_bed:
```

Comment:
Better would be to keep the naming policy and start with a verb, e.g., `convert_`